test/EFCore.PG.FunctionalTests/Query/UnaccentQueryNpgsqlTest.cs

```diff
@@ -0,0 +1,143 @@
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.TestUtilities;
+using Npgsql.EntityFrameworkCore.PostgreSQL.TestUtilities;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL.Query
+{
+    /// <summary>
+    /// Provides unit tests for the unaccent module operator and function translations.
+    /// </summary>
+    /// <remarks>
+    /// See: https://www.postgresql.org/docs/current/unaccent.html
+    /// </remarks>
+    public class UnaccentQueryNpgsqlTest : IClassFixture<UnaccentQueryNpgsqlTest.UnaccentQueryNpgsqlFixture>
+    {
+        UnaccentQueryNpgsqlFixture Fixture { get; }
+
+        public UnaccentQueryNpgsqlTest(UnaccentQueryNpgsqlFixture fixture, ITestOutputHelper testOutputHelper)
+        {
+            Fixture = fixture;
+            Fixture.TestSqlLoggerFactory.Clear();
+            //Fixture.TestSqlLoggerFactory.SetTestOutputHelper(testOutputHelper);
+        }
+
+        #region FunctionTests
+
+        [Fact]
+        public void Unaccent()
+        {
+            using var context = CreateContext();
+            var _ = context.UnaccentTestEntities
+                .Select(x => EF.Functions.Unaccent(x.Text))
+                .ToArray();
+
+            AssertContainsSql(@"unaccent(u.""Text"")");
+        }
+
+        [Fact]
+        public void Unaccent_with_regdictionary()
```

Comment:
Can you please add a Unaccent_with_parameter_regdictionary (and rename this to Unaccent_with_constant_regdictionary), where the dictionary is an external variable? I strongly suspect that test would fail, since the new .NET function accepts a string, but the actual PG function accepts a regdictionary. Take a look at how we deal with this in [the full-text search functions](https://github.com/npgsql/EFCore.PG/blob/7fee80c8e96a2314ed62d011d1e3e20f17189728/src/EFCore.PG/Query/ExpressionTranslators/Internal/NpgsqlFullTextSearchMethodTranslator.cs#L283), where we have an almost identical situation with regconfig

Reply:
yes, you were right, it didn't work properly with a parameter. So I followed what was done for the regconfig. Also, since the Unaccent methods are now in the NpgsqlFullTextSearchDbFunctionsExtensions file, I moved the tests to FullTextSearchDbFunctionsNpgsqlTest